src/SprykerEco/Zed/Computop/ComputopDependencyProvider.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Computop;
+
+use Spryker\Zed\Kernel\AbstractBundleDependencyProvider;
+use Spryker\Zed\Kernel\Container;
+use SprykerEco\Zed\Computop\Dependency\Facade\ComputopToCalculationBridge;
+use SprykerEco\Zed\Computop\Dependency\Facade\ComputopToComputopServiceBridge;
+use SprykerEco\Zed\Computop\Dependency\Facade\ComputopToMessengerBridge;
+use SprykerEco\Zed\Computop\Dependency\Facade\ComputopToOmsBridge;
+use SprykerEco\Zed\Computop\Dependency\Facade\ComputopToSalesBridge;
+
+class ComputopDependencyProvider extends AbstractBundleDependencyProvider
+{
+    const COMPUTOP_SERVICE = 'computop_service';
```

Comment:
`const SERVICE_COMPUTOP = 'computop service';`

Reply:
We usually also just repeat the key as value as this is simpler.